site2/tools/pulsar-admin-doc-gen.sh

```diff
@@ -32,11 +32,17 @@ $ROOT_DIR/bin/pulsar-admin documents generate broker-stats > $ROOT_DIR/site2/web
 $ROOT_DIR/bin/pulsar-admin documents generate brokers > $ROOT_DIR/site2/website/brodocs/documents/brokers.md
 $ROOT_DIR/bin/pulsar-admin documents generate clusters > $ROOT_DIR/site2/website/brodocs/documents/clusters.md
 $ROOT_DIR/bin/pulsar-admin documents generate functions > $ROOT_DIR/site2/website/brodocs/documents/functions.md
+$ROOT_DIR/bin/pulsar-admin documents generate functions-worker > $ROOT_DIR/site2/website/brodocs/documents/functions-worker.md
 $ROOT_DIR/bin/pulsar-admin documents generate namespaces > $ROOT_DIR/site2/website/brodocs/documents/namespaces.md
 $ROOT_DIR/bin/pulsar-admin documents generate ns-isolation-policy > $ROOT_DIR/site2/website/brodocs/documents/ns-isolation-policy.md
 $ROOT_DIR/bin/pulsar-admin documents generate sources > $ROOT_DIR/site2/website/brodocs/documents/sources.md
 $ROOT_DIR/bin/pulsar-admin documents generate sinks > $ROOT_DIR/site2/website/brodocs/documents/sinks.md
 $ROOT_DIR/bin/pulsar-admin documents generate topics > $ROOT_DIR/site2/website/brodocs/documents/topics.md
+$ROOT_DIR/bin/pulsar-admin documents generate non-persistent > $ROOT_DIR/site2/website/brodocs/documents/non-persistent.md
+$ROOT_DIR/bin/pulsar-admin documents generate persistent > $ROOT_DIR/site2/website/brodocs/documents/persistent.md
+$ROOT_DIR/bin/pulsar-admin documents generate proxy-stats > $ROOT_DIR/site2/website/brodocs/documents/proxy-stats.md
+$ROOT_DIR/bin/pulsar-admin documents generate resourcegroups > $ROOT_DIR/site2/website/brodocs/documents/resourcegroups.md
+$ROOT_DIR/bin/pulsar-admin documents generate transactions > $ROOT_DIR/site2/website/brodocs/documents/transactions.md
```

Comment:
The transaction does not seem to support this operation yet

Reply:
Hi, @congbobo184 , could you help to confirm if this is supported?

Reply:
Confirm by @congbobo184 . Already supported.